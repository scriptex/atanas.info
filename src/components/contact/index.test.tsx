import * as React from 'react';
import { render, waitFor } from '@testing-library/react';

import { Contact } from '.';

describe('Contact component', () => {
	it('Should render the Contact component', async () => {
		const { asFragment } = await waitFor(() => render(<Contact />));

		expect(asFragment()).toMatchSnapshot();
	});
});
