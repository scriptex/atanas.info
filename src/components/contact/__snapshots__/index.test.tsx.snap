// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Contact component Should render the Contact component 1`] = `
<form
  action="https://formspree.io/meqrwbvd"
  class="c-contact"
  method="POST"
>
  <button
    class="c-contact__close"
    type="button"
  >
    Close
  </button>
  <div
    class="c-contact__body"
  >
    <h2>
      Send me your message
    </h2>
    <div
      class="c-contact__field"
    >
      <label
        for="email"
      >
        Email:
      </label>
      <input
        id="email"
        name="email"
        required=""
        type="email"
      />
    </div>
    <div
      class="c-contact__field"
    >
      <label
        for="message"
      >
        Message:
      </label>
      <textarea
        cols="30"
        id="message"
        name="message"
        required=""
        rows="10"
      />
    </div>
    <div
      class="c-contact__actions"
    >
      <button
        class="c-btn "
        type="submit"
      >
        Send message
      </button>
    </div>
  </div>
</form>
`;
